deepfence_worker/worker.go

```diff
@@ -238,28 +88,19 @@ func startWorker(wml watermill.LoggerAdapter, cfg config) error {
 	ingestC := make(chan *kgo.Record, 10000)
 	go utils.StartKafkaProducer(ctx, cfg.KafkaBrokers, ingestC)
 
-	// task publisher
-	publisher, err := kafka.NewPublisher(
-		kafka.PublisherConfig{
-			Brokers:   cfg.KafkaBrokers,
-			Marshaler: kafka.DefaultMarshaler{},
+	mux := asynq.NewServeMux()
+
+	srv := asynq.NewServer(
+		asynq.RedisClientOpt{Addr: cfg.RedisAddr},
+		asynq.Config{
+			Concurrency: 10,
```

Comment:
Can you add ErrorHandler, Logger?
https://github.com/hibiken/asynq/blob/master/server.go#L155
```
	srv := asynq.NewServer(
		asynq.RedisClientOpt{Addr: cfg.RedisAddr},
		asynq.Config{
			Concurrency: 20,
			Queues: map[string]int{
				"critical": 6,
				"default":  3,
				"low":      1,
			},
			StrictPriority: false,
			ErrorHandler:   nil,
			Logger:         nil,
			LogLevel:       0,
		},
	)
```